src/App.js

```diff
@@ -1,25 +1,45 @@
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import Movies from './Movies';
+import SingleMovie from './SingleMovie';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      singleMovie: false,
+      error: '',
+    }
+  };
+
+  componentDidMount = () => {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => response.json())
+      .then(data => this.setState({movies: data.movies}))
+      .catch(error => this.setState({error: error}));
+  };
+
+  showSingleMovie = (id) => {
```

Comment:
This definitely works!

If you haven't already, consider how you might utilize the API to get this information. The reason is that when you implement router, a common pattern is to call for the single movie so that if a user directly visits a single movie link (such as yourapp.com/movies/3467, where the number is the id of one specific movie), they can see that movie without any issue as it is not dependent on it's parent component for data. I don't think this pattern is required - but it's something to consider.